### PR TITLE
Add support for Pine64 SOQuartz CM4 Module based on Quartz64 files.

### PR DESCRIPTION
**Pull request by gillham**
With the device tree blob file it boots a Linux kernel successfully and finds devices.

### CODE REVIEW

**Comment by gillham** [comment]
Linux crashes without the dtb.  I'm guessing that means UEFI / ACPI isn't working right.

Working:
USB flash drive (grub efi)
USB keyboard
Ethernet
Wifi

Partially working:
HDMI (Works in UEFI, kernel doesn't initialize it)
eMMC for Linux root (UEFI doesn't see)
SD for Linux root (UEFI doesn't see)

With the Pine64 recommended kernel (linked here: https://wiki.pine64.org/wiki/SOQuartz_Software_Releases) most devices are found.

I'm using Debian 11 and the Pine64 kernel.  Basically installed via the SOQuartz debian installer and upgraded to 11.

I have also tested the Debian Experimental kernel (5.16.0-rc7) which finds the Ethernet and SD cards so can boot up, but not USB.  So all testing is generally with the Pine64 kernel.

Using something like this with grub:
linux /boot/vmlinuz root=/dev/mmcblk0p5 ro console=ttyS2,115200
devicetree /boot/dtbs/rockchip/rk3566-soquartz-cm4.dtb
boot

**Comment by gillham** [comment]
Ok, I missed the obvious regarding the .dtb file getting included into uefi.its.  I reworked that slightly to build separate files.

File: Makefile [hunk 1]
```diff
@@ -18,25 +18,34 @@ sdcard: uefi_release
...
+	gzip sdcard_*_?GB.img
 
 .PHONY: clean
 clean:
 	rm -rf Build
 	rm -f bl31_*.bin
 	rm -f idblock.bin
-	rm -f QUARTZ64_EFI_*.itb
+	rm -f SOQuartz_EFI_*.itb
+	rm -f Quartz64_EFI_*.itb
```
Comment (jaredmcneill) [inline Makefile]: Can you keep these uppercase please?

Comment (gillham) [reply Makefile]: Just the `*.itb` files or do you want uppercase in the `sdcard_*` files also?

Comment (jaredmcneill) [reply Makefile]: Only really concerned about the itb files. No opinion on `sdcard_*` so do whatever is easiest there.

Comment (gillham) [reply Makefile]: In `build_fit()` we need the correct case for `Build/${board}` but we can use uppercase (`${board_upper}` perhaps) for 
`${board_upper}_EFI_*.itb`
`uefi_${board_upper}.its`
`sdcard_${board_upper}_*.img`

Comment (gillham) [reply Makefile]: Ok, I adjusted the Makefile & build.sh to uppercase the `*.itb` files only.

**Pull request merged**